shared/macros-bash.jinja

```diff
@@ -361,8 +331,10 @@ for firefox_dir in ${firefox_dirs}; do
     if [ -d "${firefox_dir}" ]; then
         # Make sure the Firefox .cfg file exists and has the appropriate permissions
         if ! [ -f "${firefox_dir}/${firefox_cfg}" ] ; then
-            touch "${firefox_dir}/${firefox_cfg}"
+            echo "//" "${firefox_dir}/${firefox_cfg}"
```

Comment:
@cyarbrough76 why this change?

Reply:
These changes facilitate aligning the remediation with the latest DISA STIG and benchmark.

While ideally, the file name really shouldn't matter, but since that's what the DISA benchmark specifically looks for. It's easier just to pass a scan than explain for each failure how it's a pass.

The changes for adding a '//' in the first line of the file is because the first line of the file is ignored. This is also outlined in the STIG.

The file path changes were made to give the default preference the location the STIG looks for.

The variables at the top also needed a slash to properly resolve.